packages/x-date-pickers/src/locales/deDE.ts

```diff
@@ -60,13 +60,13 @@ const deDEPickers: Partial<PickersLocaleText<any>> = {
   secondsClockNumberText: (seconds) => `${seconds}  ${clockViews.seconds}`,
 
   // Open picker labels
-  openDatePickerDialogue: (rawValue, utils) =>
-    rawValue && utils.isValid(utils.date(rawValue))
-      ? `Datum auswählen, gewähltes Datum ist ${utils.format(utils.date(rawValue)!, 'fullDate')}`
+  openDatePickerDialogue: (value, utils) =>
+    value != null && utils.isValid(value)
```

Comment:
Nitpick: Shouldn't it be `value !== null` instead? 🤔

Reply:
It should not change anything here since the value can't be undefined
But yes we should have the strict test

Fixed